src/cmd/services/m3dbnode/config/fetch_tagged.go

```diff
@@ -25,5 +25,15 @@ package config
 type FetchTaggedConfiguration struct {
 	// SeriesBlocksPerBatch specifies how many series blocks are allowed to be retrieved
 	// per permit acquired. Defaults to 1 (i.e. a permit is acquired per series block).
-	SeriesBlocksPerBatch int `yaml:"seriesBlocksPerBatch" validate:"min=0"`
+	SeriesBlocksPerBatch *int `yaml:"seriesBlocksPerBatch" validate:"min=1"`
```

Comment:
I recently added this config earlier this week, so this should be safe to modify